templates/user/dashboard/feeds.tmpl

```diff
@@ -1,4 +1,11 @@
 {{range .Feeds}}
+	{{ $titleName := .GetActFullName }}
+	{{ $titleNameValue := .ShortActUserName }}
+	{{ if and DefaultShowFullName (ne .GetActFullName "") }}
+		{{ $titleName = .ShortActUserName }}
```

Comment:
This should also be moved to models as this code won't work in golang 1.9

Reply:
what's the reason to still support such an old version? 1.9 doensn't get security updates anymore.

Reply:
that is not the only reason, this logic would be also better in methods as when per-user preference will be added this would have to be rewritten again

Reply:
Ok, fixed by using 2 extra helper functions